src/Methods/Modular_Programming/Hard/Q1/MainTest.java

```diff
@@ -0,0 +1,56 @@
+package Methods.Modular_Programming.Hard.Q1;
+import global.BaseTest;
+import global.tools.TestOption;
+import global.variables.*;
+import global.variables.clauses.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MainTest extends BaseTest{
+
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "5";
+
+        return new Clause[]{
+                new StringLiteral("Enter a number"),
+                new NewLine(),
+                new StringLiteral("true"),
```

Comment:
Why is this always true?

Reply:
Because this program uses a method to do the calculations, I only really run the clause for the default input, which should return true. So it doesn't need to be anything but true.